src/main/resources/application.properties

```diff
@@ -1 +1,34 @@
 spring.application.name=ez-quiz-backend
+
+# HTTP port
+server.port=8080
+
+# Database Configuration for H2 Database
+spring.datasource.url=jdbc:h2:mem:testdb;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;AUTO_RECONNECT=TRUE;MODE=PostgreSQL;
```

Comment:
You will want to hide url, username, and password on this file for your datasources